requirements.txt

```
@@ -25,7 +25,7 @@ tensorboard
 jinja2==3.0.3
 pytorch-lightning
 torchx
-torchrl==0.2.0
+torchrl==0.2.1
```

```suggestion
torchrl==0.2.0
```
we're sticking to 0.2.0 for now, or we need to upgrade both rl and tensordict to 0.2.1

---

Does this work on MacOS with 0.2.1?

---

yes it should so to me we can update both dependencies